DWM/update_abos_attributes_to_dwm.py

```diff
@@ -1,51 +1,43 @@
 #!/usr/bin/env python3
 """Rename ABOS files and attributes"""
+# As part of the ABOS renaming Odyssey, this script identifies the attributes that contains the words "ABOS" and
+# "Bluewater" (as art of the Australian Bluewater Observation System), changes them for "DWM"/"Deep Water Moorings"
+# and rename the files. The attribute 'history' is excluded from the search as we decide that attribute did not need
+# to be changed.
```

Comment:
Maybe we don't need to refer to the "ABOS renaming odyssey" in a public repo...
Also, you can make this paragraph part of the "docstring" for the script, by including it in the triple-quoted text starting on line 2:
```python
"""Rename ABOS files and attributes

This script identifies the attributes...
"""
```
Also a couple of typos in there (contain~~s~~, art -> part)